### ISSUE TITLE
Craft 5 support

### ISSUE THREAD

**Issue by mihob** [panlatent/schedule#26]
Just checking, do you have plans for craft 5 support?

**Comment by panlatent** [comment]
The latest code already supports Craft 5, but it hasn't been tested enough so it hasn't been released yet.

**Comment by panlatent** [comment]
@mihob  v0.6.0 released!

**Comment by mihob** [comment]
Awesome! Thx!

**Comment by plcdnl** [comment]
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - panlatent/schedule 0.1.1 requires craftcms/cms ^3.0.0 -> found craftcms/cms[3.0.0-beta.1, ..., 3.x-dev] but it conflicts with your root composer.json require (5.1.2).
    - panlatent/schedule[0.1.2, ..., 0.3.2] require craftcms/cms ^3.1.0 -> found craftcms/cms[3.1.0-beta.1, ..., 3.x-dev] but it conflicts with your root composer.json require (5.1.2).
    - panlatent/schedule[dev-generics, 0.4.0, ..., 0.5.0] require craftcms/cms ^4.0 -> found craftcms/cms[4.0.0-alpha.1, ..., 4.x-dev] but it conflicts with your root composer.json require (5.1.2).
    - panlatent/schedule[dev-master, 0.4.x-dev, ..., 0.6.0] require nesbot/carbon ^1.22 || ^2.10 -> found nesbot/carbon[1.22.0, ..., 1.39.1, 2.10.0, ..., 2.x-dev] but the package is fixed to 3.3.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires panlatent/schedule * -> satisfiable by panlatent/schedule[dev-master, dev-generics, 0.1.1, ..., 0.6.0].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require panlatent/schedule:*" to figure out if any version is installable, or "composer require panlatent/schedule:^2.1" if you know which you need.
```

Got this error when i try to install the plugin. Why ?

**Comment by panlatent** [comment]
> ```
> Your requirements could not be resolved to an installable set of packages.
> 
>   Problem 1
>     - panlatent/schedule 0.1.1 requires craftcms/cms ^3.0.0 -> found craftcms/cms[3.0.0-beta.1, ..., 3.x-dev] but it conflicts with your root composer.json require (5.1.2).
>     - panlatent/schedule[0.1.2, ..., 0.3.2] require craftcms/cms ^3.1.0 -> found craftcms/cms[3.1.0-beta.1, ..., 3.x-dev] but it conflicts with your root composer.json require (5.1.2).
>     - panlatent/schedule[dev-generics, 0.4.0, ..., 0.5.0] require craftcms/cms ^4.0 -> found craftcms/cms[4.0.0-alpha.1, ..., 4.x-dev] but it conflicts with your root composer.json require (5.1.2).
>     - panlatent/schedule[dev-master, 0.4.x-dev, ..., 0.6.0] require nesbot/carbon ^1.22 || ^2.10 -> found nesbot/carbon[1.22.0, ..., 1.39.1, 2.10.0, ..., 2.x-dev] but the package is fixed to 3.3.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
>     - Root composer.json requires panlatent/schedule * -> satisfiable by panlatent/schedule[dev-master, dev-generics, 0.1.1, ..., 0.6.0].
> 
> Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
> You can also try re-running composer require with an explicit version constraint, e.g. "composer require panlatent/schedule:*" to figure out if any version is installable, or "composer require panlatent/schedule:^2.1" if you know which you need.
> ```
> 
> Got this error when i try to install the plugin. Why ?当我尝试安装插件时出现此错误。为什么 ？

Craft 5 requires version 0.6.0 of this plugin, the package does not yet support the 3.x version of `nesbot/carbon`, you appear to be locked to 3.x.

**Comment by plcdnl** [comment]
Yes, the problem is nesbot/carbon. For "feed me plugin" version 3.x is required. I am sorry

**Comment by panlatent** [comment]
Thank! I think compatibility with feedme is necessary. But it still needs some time to test, the fixed version will be released in 0.6.1
